### PR TITLE
Updated All Packages (mostly to get to latest Nuxt)

### PR DESCRIPTION
**Pull request by buffet-time**
## What

updated Nuxt to latest, and all other packages, ensured it built & tests passed

### CODE REVIEW

**Review by zysim — APPROVED** [review]

**Review by erunks — APPROVED** [review]

**Comment by TheTedder** [comment]
Can we close this?

**Comment by zysim** [comment]
Guess not with the conflicts now. I'll tidy them up

**Comment by zysim** [comment]
Fixed conflicts, but I'm now getting a bunch of warnings and non-test-failing errors when I run tests. A bunch of em in particular are on `<RouterLink>` not being found. Not sure how to fix that right now.

File: package.json [hunk 1]
```diff
@@ -54,11 +54,10 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-tailwindcss": "^3.17.4",
-    "eslint-plugin-vue": "^9.28.0",
+    "eslint-plugin-vue": "^9.27.0",
```
Comment (TheTedder) [inline package.json:57]: Why are we downgrading this?

File: package.json [hunk 2]
```diff
@@ -54,11 +54,10 @@
...
     "eslint-plugin-vuejs-accessibility": "^2.4.1",
-    "eslint-typegen": "^0.3.1",
```
Comment (TheTedder) [inline package.json:59]: Why is this dep being removed?

File: package.json (hunk 1 above)
Comment (zysim) [reply package.json:57]: Mistake

**Pull request merged**